os/fs/smartfs/smartfs_utils.c

```diff
@@ -1036,6 +1036,65 @@ int smartfs_unmount(struct smartfs_mountpt_s *fs)
 	return ret;
 }
 
+/****************************************************************************
+ * Name: smartfs_get_datalen
+ *
+ * Description: Calculates the length of the opened file
+ *
+ ****************************************************************************/
+
+int smartfs_get_datalen(struct smartfs_mountpt_s *fs, uint16_t firstsector, uint32_t *datalen)
+{
+	int ret;
```

Comment:
It is better to initialize the `ret`, because, some cases `ret` will not be assigned, but will be checked in linke 1090.
Then `ret` can be compared with trash value.

Reply:
Assigned, thanks